cosmos-cmd-tlm-api/spec/controllers/metadata_controller_spec.rb

```diff
@@ -0,0 +1,200 @@
+# encoding: ascii-8bit
+
+# Copyright 2022 Ball Aerospace & Technologies Corp.
+# All Rights Reserved.
+#
+# This program is free software; you can modify and/or redistribute it
+# under the terms of the GNU Affero General Public License
+# as published by the Free Software Foundation; version 3 with
+# attribution addendums as found in the LICENSE.txt
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# This program may also be used under the terms of a commercial or
+# enterprise edition license of COSMOS if purchased from the
+# copyright holder
+
+require 'rails_helper'
+
+RSpec.describe MetadataController, :type => :controller do
+  before(:each) do
+    mock_redis()
+  end
+
+  describe "POST create" do
+    it "returns a hash and status code 201" do
```

Comment:
The it "whatever" lines should talk about what is being tested, not the data being returned.

Reply:
returning a hash and the status code is about all I test...